src/pdfflow/pflow.py

```diff
@@ -191,7 +193,7 @@ def __init__(self, dirname, fname, members, compilable=True):
 
         for member_int in members:
             member = str(member_int).zfill(4)
-            filename = f"{self.dirname}/{fname}/{fname}_{member}.dat"
+            filename = os.path.join(self.dirname, fname, f'{fname}_{member}.dat')
```

Comment:
I don't understand this. Was the previous version failing? (Using os.path.join is fine and actually better if this is just a cosmetic change)

```suggestion
            filename = os.path.join(self.dirname, fname, f"{fname}_{member}.dat")
```

Reply:
No, it wasn't failing. That's just because the log I was getting looked bad: multiple // in the string like `'dirname//fname/fname_member.dat'`